data/api/client-server/profile.yaml

```diff
@@ -98,6 +98,20 @@ paths:
                   value: {
                     "displayname": "Alice Margatroid"
                   }
+        "403":
+          x-addedInMatrixVersion: "1.2"
```

Comment:
I don't know if this is correct but technically we're clarifying the 403 response introduced in Matrix v1.2.